x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -346,6 +346,7 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
           const populatedException = getPrepopulatedRuleExceptionWithHighlightFields({
             alertData,
             exceptionItemName,
+            ruleCustomHighlightedFields: rules?.[0]?.investigation_fields ?? [],
```

Comment:
Removed! Thanks.

Reply:
We only pre-populate when adding from an alert, in which case there's just one rule associated. But that was a good call - I can leave a comment so that that is clear here and also something to consider if we end up changing this code in the future.